wurst/_handles/primitives/String.wurst

```diff
@@ -251,6 +253,58 @@ public function string.format(string par1, string par2, string par3, string par4
 				.replace("{8}", par9)
 				.replace("{9}", par10)
 				
+/** Replaces consecutive repeats of the same string with a single instance 
+	Turns "strstr" into "str" */
+public function string.reduce(string reduced) returns string
+	string toReduce = reduced + reduced 
+	string oldString = this
+	string newString
+
+	while true
```

Comment:
There's an `O(n)` solution for this but not sure how much faster in practice. I think this is fine.